tsl/test/sql/bgw_reorder_drop_chunks.sql

```diff
@@ -201,7 +201,7 @@ SELECT ts_bgw_params_reset_time();
 \c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
 
 -----------------------------------
--- test drop chunnks policy runs --
+-- test drop chunks policy runs --
 -----------------------------------
```

Comment:
I probably should cut some `-`s from the surrounding items...

I mostly haven't tried to do this. I did for the test run output where the tests failed...